content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,89 @@ description: Istio 1.5 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Improved** performance of `ServiceEntry` updates #19305.
```

Comment:
should we have a dedicated performance section?  Having a line for performance improvement for 1 thing doesn't really help users.